src/utils/utils/headless-service.ts

```diff
@@ -0,0 +1,31 @@
+import { ServiceModel } from '@kubevirt-ui/kubevirt-api/console';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+
+export const HEADLESS_SERVICE_LABEL = 'app.kubernetes.io/name';
+export const HEADLESS_SERVICE_NAME = 'headless';
+export const HEADLESS_SERVICE_PORT = 5434;
+
+export const createHeadLessService = (createdVM: V1VirtualMachine) => {
+  try {
+    k8sCreate({
+      data: {
+        apiVersion: 'v1',
+        kind: 'Service',
```

Comment:
ServiceModel is already imported, you can use ServiceModel.apiVersion and .kind instead of hard coded values